code/modules/jobs/job_types/clown.dm

```diff
@@ -62,7 +62,8 @@
 	if(visualsOnly)
 		return
 
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/woodenducky)
```

Comment:
Stop trying to teach dead clowns how to craft wooden duckys.

We spawn dead mind-less clowns on lavaland